prow/config.yaml

```diff
@@ -122,6 +122,11 @@ presets:
     env:
       - name: DOCKER_PUSH_REPOSITORY
         value: "eu.gcr.io/kyma-project"
+  - labels:
```

Comment:
Why do we have a duplicated label here, when `preset-docker-push-repository` is adding the exact same value?

Reply:
I see why from looking at the template below. Added a comment there.